CryptoTechReminderSystem/UseCase/SendReminder.cs

```diff
@@ -21,10 +21,12 @@ public void Execute(SendReminderRequest sendReminderRequest)
                 Channel = sendReminderRequest.Channel,
                 Text = sendReminderRequest.Text
             });
+
+            var address = sendReminderRequest.Email ?? sendReminderRequest.Channel;
             
-            result.OnSuccess(success => Console.WriteLine($"{sendReminderRequest.Email} was sent a reminder."));
+            result.OnSuccess(success => Console.WriteLine($"{address} was sent a reminder."));
             result.OnError(error => Console.WriteLine(
-                $"!Failed to send message to {sendReminderRequest.Email} with error: {error.Message}"));
+                $"!Failed to send message to {address} with error: {error.Message}"));
```

Comment:
Is the `!` in the string meant to be there?

Reply:
Yes, just to mark it as an error message, but not sure about this, happy to remove